temporalcli/payload.go

```diff
@@ -0,0 +1,29 @@
+package temporalcli
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"go.temporal.io/api/common/v1"
+)
+
+func CreatePayloads(data [][]byte, metadata map[string][]byte, isBase64 bool) (*common.Payloads, error) {
```

Comment:
```suggestion
func createPayloads(data [][]byte, metadata map[string][]byte, isBase64 bool) (*common.Payloads, error) {
```

Probably don't need to export (though it doesn't really matter in the context of an application vs a library so there's no rules here, so nothing required to change)

Reply:
OK thanks, my Go style is a work-in-progress... when I uppercased it I was thinking that I was working in a separate "util" package from the main commands, but as you point out that's not the case.